instrumentation/aws-sdk/aws-sdk-1.11/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v1_11/RequestAccess.java

```diff
@@ -90,6 +150,12 @@ private RequestAccess(Class<?> clz) {
     getTableName = findAccessorOrNull(clz, "getTableName");
     getTopicArn = findAccessorOrNull(clz, "getTopicArn");
     getTargetArn = findAccessorOrNull(clz, "getTargetArn");
+    getStateMachineArn = findAccessorOrNull(clz, "getStateMachineArn");
+    getStepFunctionsActivityArn = findAccessorOrNull(clz, "getActivityArn");
+    getSnsTopicArn = findAccessorOrNull(clz, "getTopicArn");
+    getSecretArn = findAccessorOrNull(clz, "getARN");
+    getLambdaName = findAccessorOrNull(clz, "getFunctionName");
+    getLambdaResourceId = findAccessorOrNull(clz, "getUUID");
```

Comment:
I couldn't find it being used outside of lambda sdk, but given its generic name it could benefit from an extra check to ensure that it is not added to some random request